apps/billing/payments/zibal_client.py

```
@@ -17,14 +15,18 @@
 
 class ZibalConfig:
     """Zibal configuration."""
-    
+
+    DEFAULT_GATEWAY_BASE = 'https://gateway.zibal.ir'
+
     def __init__(self):
```

Ruff (0.14.0)

Missing return type annotation for special method `__init__`

Add return type annotation: `None`

(ANN204)